docs/source/api/ApolloCodegenLib/classes/EnumGenerator.md

```diff
@@ -0,0 +1,58 @@
+**CLASS**
+
+# `EnumGenerator`
+
+```swift
+public class EnumGenerator
+```
+
+## Properties
+### `enumTemplate`
+
+```swift
+open var enumTemplate = """
```

Comment:
Looks like our doc generator will put the full value of a directly assigned variable into the docs. Good to know 😛